### PR TITLE
net.minecraft.item coverage

### PR DESCRIPTION
**Pull request by moehreag**
Completes coverage in the net.minecraft.item package. This is rather small since most classes were already completely mapped.

*I hope there aren't too many things that need to be changed..*

*Of course rai was faster already with a few of these changes in #373  :p*

### CODE REVIEW

**Comment by eristhea** [comment]
> Of course rai was faster already with a few of these changes in https://github.com/QuiltMC/quilt-mappings/pull/373 :p

<3

**Review by eristhea — APPROVED** [review]
brilliant!

File: mappings/net/minecraft/item/VanishableArmorItem.mapping [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/unmapped/C_kelbxmwg net/minecraft/item/VanishableArmorItem
```
Comment (eristhea) [inline mappings/net/minecraft/item/VanishableArmorItem.mapping]: actually one thing.
this seems like *way* too specific of a name, given that all equipment that can be worn can have vanishing curse
could we not just name this something like `Equippable` or `Equipment`?
probably the longest name I'd be ok with is `WearableEquipment`

**Review by eristhea — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/item/VanishableArmorItem.mapping (hunk 1 above)
Comment (OroArmor) [reply mappings/net/minecraft/item/VanishableArmorItem.mapping]: I think this used to be `Vanishable`

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/item/VanishableArmorItem.mapping (hunk 1 above)
Comment (moehreag) [reply mappings/net/minecraft/item/VanishableArmorItem.mapping]: No, but this *extends* `Vanishable` and is only used by ArmorItems, Skulls, Carved Pumpkins, Elytras and Shields (which are basically all items that can be worn).

Comment (OroArmor) [reply mappings/net/minecraft/item/VanishableArmorItem.mapping]: Ah ok. Then i would say `WearableVanishable` or the other way around

Comment (OroArmor) [reply mappings/net/minecraft/item/VanishableArmorItem.mapping]: I hate that so much though

Comment (eristhea) [reply mappings/net/minecraft/item/VanishableArmorItem.mapping]: yeah no that's a bad name
I checked mojang and they simply call it `Equippable` so I think it's best we match that

Comment (moehreag) [reply mappings/net/minecraft/item/VanishableArmorItem.mapping]: sure!

**Review by eristhea — APPROVED** [review]

**Review by EnnuiL — APPROVED** [review]

**Pull request merged**